app/views/players/_form.html.slim

```diff
@@ -63,7 +63,11 @@
             input_html: { class: 'mr-1' },
             wrapper_html: { class: 'align-middle mr-3' }
     - if @tournament.manual_seed
-      = f.input :manual_seed, placeholder: 'Set seed', inline_label: 'Manual Seed', required: false, as: :string,
+      = f.input :manual_seed, placeholder: 'Set seed', label: 'Manual Seed', required: false, as: :string,
               wrapper_html: { class: 'form-group my-2' },
-              input_html: { class: 'form-control manual_seed' },
+              input_html: { class: 'form-control manual-seed' },
               label_html: { class: 'align-middle d-inline-block mr-1' }
+    = f.input :fixed_table_number, placeholder: 'Enter table number', label: 'Table Number', required: false, as: :string,
```

Comment:
The only UI suggestion i would make it to get this onto the same row as the Save/Drop/Delete options if possible.

Reply:
It's pretty awkward the way it's set up right now. It might be worth doing that separately.

Reply:
Yep, sounds good.